testing/sdk_hosts.py

```diff
@@ -35,6 +36,19 @@ def vip_host(service_name, vip_name, port=-1):
     return _to_host(_safe_name(vip_name), _safe_name(service_name), VIP_HOST_SUFFIX, port)
 
 
+def scheduler_vip_host(service_name, vip_name, port=-1):
```

Comment:
Nit: The `vip_name` comes first in the resultant `vip`. Not a blocker.

Reply:
I didn't like it too much too but I decided to make it consistent with `vip_host`.